microcosm_eventsource/tests/fixtures.py

```diff
@@ -269,6 +329,57 @@ def event_factory(self):
         )
 
 
+@binding("immutable_task_event_controller")
+class ImmutableTaskEventController(EventController):
+    def __init__(self, graph):
+        super(ImmutableTaskEventController, self).__init__(
+            graph,
+            store=graph.immutable_task_event_store
+        )
+        self.ns = Namespace(
+            subject=ImmutableTaskEvent,
+            version="v1",
+        )
+        self.immutable_task_event_factory = graph.immutable_task_event_factory
+
+    def get_event_factory_kwargs(self):
+        return {}
+
+    @property
+    def event_factory(self):
+        return self.immutable_task_event_factory
+
+
+@binding("immutable_task_event_factory")
+class ImmutableTaskEventFactory(ImmutableEventFactory):
+    def __init__(self, graph):
+        super().__init__(
+            event_store=graph.immutable_task_event_store,
+            container_store=graph.immutable_task_store,
+            identifier_key="immutable_task_event_id",
+            default_ns=Namespace(
+                subject=ImmutableTaskEvent,
+                version="v1",
+            ),
+        )
+
+    @register_container_mutator_common(event_type=ImmutableTaskEvent)
```

Comment:
common event handler for all events

Reply:
It really feels like we could use a more concise name, e.g. `common_mutator`? The action `register` is implied? By decorating something as a `common_mutator` we are registering it as a `common_mutator`. Also, I think the design of how we are doing aggregation implies we are aggregating state on the `container` so again adding that in the decorator name is redundant.

Reply:
Or another naming idea, simply make the decorator  called `aggregates_event_type` and let it select the correct registry based on the type based, e.g. an event instance type or an event_type?

Reply:
How about below names?
1. common_container_mutator
2. event_specific_container_mutator

I am still inclined to keep separate decorators, for explicitness.